pages/linux/nordvpn.md

```diff
@@ -0,0 +1,24 @@
+# nordvpn
+
+> Commandline interface to NordVPN.
```

Comment:
```suggestion
> Command-line interface to NordVPN.
```

Reply:
Also, instead of `to`, `for` works better.